include/rocksdb/options.h

```diff
@@ -374,6 +374,14 @@ struct DBOptions {
   // Default: true
   bool paranoid_checks = true;
 
+  // If true, check on-disk WALs against WAL information stored in MANIFEST
+  // during recovery.
+  // It checks whether there are missing or corrupted WALs.
+  // No matter whether this is true or false, the WAL information are always
+  // tracked by MANIFEST.
+  // Default: false
+  bool check_wal = false;
```

Comment:
Why wouldn't this be true by default if we always pay the ongoing cost of tracking, and the only check is during recovery?

When / why would someone want to use false?

Reply:
If the user does not even sync WALs, then the user might skip checking the WALs since there is no guarantee of WAL's existence or completeness on recovery.